src/context/BrowserRouterProvider/BrowserRouterProvider.tsx

```diff
@@ -34,8 +34,9 @@ export function BrowserRouterProvider({ children }: BrowserRouterProviderProps)
   }, [appRoutes, location.pathname])
 
   useEffect(() => {
-    const redactedPathname = mapMixpanelPathname(location.pathname, assets)
-    getMixPanel()?.track(MixPanelEvents.PageView, { pathname: redactedPathname })
+    const processedPathname = mapMixpanelPathname(location.pathname, assets)
+    if (processedPathname !== false)
```

Comment:
We can keep the current name, perhaps prepending `maybe` if we want to make it clear this isn't necessarily going to be defined anymore, and check for `(redactedPathname)` below

Reply:
It will be defined for any event that isn't filtered by the `false` value. And now we only "redact" the `/asset/` paths but it's more for better readability than to redact anything sensitive, the paths that were "redacted" are now purely skipped... hence the variable name change. But if you prefer the old name, I can change it, please confirm.

Reply:
Done with `maybePathname`